src/config/queryClient.js

```diff
@@ -15,7 +15,7 @@ const {
   ReactQueryDevtools,
   API_ROUTES,
   MUTATION_KEYS,
-  HOOK_KEYS,
+  QUERY_KEYS,
```

Comment:
I am wondering whether you will be using these ? Out of curiosity 😊

Reply:
Never, I guess